docs/index.html

```diff
@@ -1 +1,35 @@
-Test
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Blog Post</title>
+  </head>
+  <body>
+    <img src="https://images.unsplash.com/photo-1546074177-ffdda98d214f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1567&q=80" alt="Describe what's in the photo">
+    <h1>Blog Post Title h1</h1>
+    <span>dd.mm.yyyy 00:00</span>
+    <p>Lead Paragraph – Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+    <h2>Subtitle h2</h2>
+    <p>Paragraph – Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+    <h3>Subtitle h3</h3>
+    <ul>
+      <li>list item</li>
+      <li>list item</li>
+      <li>list item</li>
+    </ul>
+    <ol>
+      <li>list item</li>
+      <li>list item</li>
+      <li>list item</li>
+    </ol>
+    <hr> <!-- Is it okay to use this tag or should it be a div or span styled with css later on? -->
+    <div>
+      <img src="https://avatars2.githubusercontent.com/u/1894309?s=120&v=4" alt="Author portrait">
```

Comment:
Perfect fine. Just out of curiosity? Where did you get the URL from? Is that officially supported? Do you know [gravatar](https://en.gravatar.com/)? Many services support it and they pre-fill your avatar from there if you open a new account.